src/constants/default-settings.js

```diff
@@ -90,7 +83,21 @@ import {
 } from 'components/common/icons';
 
 export const KEPLER_GL_NAME = 'kepler.gl';
-export const KEPLER_GL_VERSION = 'v1.0';
+
+// __PACKAGE_VERSION__ is automatically injected by Babel/Webpack during the building process
+// Since we are injecting this during the build process with babel
+// while developing VERSION is not defined, we capture the exception and return
+// an empty string which will allow us to retrieve the latest umd version
+export const KEPLER_GL_VERSION = (() => {
+  try {
+    // eslint-disable-next-line
+    return __PACKAGE_VERSION__;
```

Comment:
babel -plugin-search-and-replace searches `StringLiteral`. Here you should return `'__PACKAGE_VERSION__'` as  a string. During build `'__PACKAGE_VERSION__'` will be reaplced to '1.0.0-0'